fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,7 +1,22 @@
-Are you tired of searching for your plastic reward card while checking out at the store? Are you looking for a free solution which will not take your information?
+Stop the search for plastic reward cards during store or webshop checkout.
+Scan barcodes for deals to your device using its camera, and keep not remembering where they are.
 
-Catima is a application which will store your barcode-based loyalty cards on your phone. The application is open source and tries to do one thing well: manage your cards!
+Drop your wallet, or keep it ultralight for valuables.
 
-New cards can be added in a snap. Either use your camera to capture the barcode, or type in the number. When the barcode is loaded at the store and displayed it can be scanned with a modern barcode scanner. (Some stores use older barcode scanners, such as flatbed scanners, instead of image scanners. These cannot read the smartphone's display. Instead, request the clerk to type in the number manually).
+With this essential everyday carry (EDC) tool you can replace useless plastic with cash.
 
-The application requires very few permissions, and never attempts to access the Internet. There is an option to backup your cards to local storage. From there you can send the backup somewhere safe.
+- Avoid spying with very few permissions. No Internet access and no ads.
+- Add cards or codes with names and customizable colours.
+- Manual code entry if there is no barcode to store, or it can't be used.
+- Import cards and codes from files, Catima, Loyalty Card Keychain, Voucher Vault, and FidMe.
+- Make a backup of all your cards and transfer it to a new device if you want.
+- Share coupons, exclusive offers, promo codes, or cards and codes using any app.
+- Dark theme and accesiblity options for vision impaired users
+- Made for everyone by the libre software community.
+- Localized hand-made translations for 20+ languages.
+- Gratis, supported by community contributions.
+- Use, study, change and share it as you wish; with all.
+- Not only Open Source. Copylefted Free Software/libre software (GPLv3+) card management.
```

Comment:
Interesting you now suddenly do use Free Software despite earlier talking about avoiding FSF phrasing. but it makes sense here for the purpose of SEO, kinda.

Reply:
That is exactly why.